nix/fetchNixpkgs.nix

```diff
@@ -3,46 +3,8 @@
 , system ? builtins.currentSystem # This is overridable if necessary
```

Comment:
Doesn't look like we need `system` here anymore

Reply:
Makes sense; removed by 7e9c5563e7aaa7360078d99da766a95ee5401824 .